lite2/store/db/db_test.go

```diff
@@ -144,7 +148,7 @@ func Test_Concurrency(t *testing.T) {
 			defer wg.Done()
 
 			dbStore.SaveSignedHeaderAndValidatorSet(
-				&types.SignedHeader{Header: &types.Header{Height: i}}, &types.ValidatorSet{})
+				&types.SignedHeader{Header: &types.Header{Height: i}}, vals)
```

Comment:
why do we need vals here?

Reply:
because proto will return an error?

Reply:
on to proto it errors when it is a nil validator set.